x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -68,116 +75,198 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
     label: i18n.translate('xpack.ingestPipelines.processors.label.append', {
       defaultMessage: 'Append',
     }),
-    description: i18n.translate('xpack.ingestPipelines.processors.description.append', {
+    typeDescription: i18n.translate('xpack.ingestPipelines.processors.description.append', {
       defaultMessage:
         "Appends values to a field's array. If the field contains a single value, the processor first converts it to an array. If the field doesn't exist, the processor creates an array containing the appended values.",
     }),
+    getDefaultDescription: ({ field, value }) =>
+      i18n.translate('xpack.ingestPipelines.processors.defaultDescription.append', {
+        defaultMessage: 'Appends "{value}" to the "{field}" field',
```

Comment:
I like to write commit messages as commands, so I personally would prefer to see this written as "Append" instead of "Appends", and the other descriptions updated similarly. This also makes the messages just a bit shorter. 🤷 I dunno, I'll defer to whatever the writers think is best. :)

Reply:
Yeah the current writeup was very WIP, just to get the general format done, I was later going to reach out to at least a few others to see if someone can do some rewriting of the actual text, as it's not currently in top shape :)

Reply:
I disagree. The ingest processor takes action on the provided input. I like to think of it in a sentence and ask the question, "What does this `<name_of_thing>` do?" For example:
>"What does this machine do?" --> "It *grinds* coffee beans."

You wouldn't say, "It *grind* coffee beans". Each of the ingest processor descriptions use this same format. For example, the [Append processor](https://www.elastic.co/guide/en/elasticsearch/reference/current/append-processor.html) "Appends one or more values to an existing array...."

cc: @gchaps to add or clarify

Reply:
++ to what @lockewritesdocs said.